client/src/main/java/com/alibaba/nacos/client/naming/remote/http/NamingHttpClientProxy.java

```diff
@@ -212,7 +211,6 @@ public ServiceInfo queryInstancesOfService(String serviceName, String groupName,
         params.put(CommonParams.NAMESPACE_ID, namespaceId);
         params.put(CommonParams.SERVICE_NAME, NamingUtils.getGroupedName(serviceName, groupName));
         params.put(CLUSTERS_PARAM, clusters);
-        params.put(UDP_PORT_PARAM, String.valueOf(udpPort));
```

Comment:
Http client request this port, if you need remove, please remove whole implementation, or keep this parameter.